charts/wa-task-management-api/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for wa-task-management-api App
 name: wa-task-management-api
 home: https://github.com/hmcts/wa-task-management-api
-version: 0.0.2
```

Comment:
No need to bump it up manually anymore because of this new config here: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/wa/wa-task-management-api/wa-task-management-api.yaml#L7 

Flux will do it for you.